### PR TITLE
Adding name instead of id to categories in tasks

### PR DESCRIPTION
**Pull request by Angamanga**
This pull request makes the following changes:
- Displaying name instead of Id to categories in tasks

Testing checklist:
- [ ] Add a post to a survey with categories in a task and save the post. The categories selected should be displayed as names, not as ids when looking at the saved post.

Fixes ushahidi/platform#2022 .

Ping @ushahidi/platform

### CODE REVIEW

File: app/main/posts/detail/detail.html [hunk 1]
```diff
@@ -116,15 +117,25 @@ <h3 class="listing-heading" translate="app.tasks">Tasks</h3>
           </div>
           <div
             ng-repeat="(key,value) in post.values"
-            ng-if="form_attributes[key].type !== 'media' && !isPostValue(key) && form_attributes[key] && showType(form_attributes[key].type)"
             ng-show="form_attributes[key].form_stage_id === visibleTask"
             >
             <post-value
+              ng-if="form_attributes[key].type !== 'media' && !isPostValue(key) && form_attributes[key] && showType(form_attributes[key].type)&& form_attributes[key].input !== 'tags'"
```
Comment (rjmackay) [inline app/main/posts/detail/detail.html]: Can we move this if condition to a function on the controller?
In actually looks like `showType` might already be doing that and the other conditions should get merged in?

File: app/main/posts/detail/detail.html [hunk 2]
```diff
@@ -116,15 +117,25 @@ <h3 class="listing-heading" translate="app.tasks">Tasks</h3>
...
               key="key"
               value="value"
               attribute="form_attributes[key]"
               type="'standard'">
             </post-value>
+             <div class="listing-item-primary" ng-if="form_attributes[key].input === 'tags'">
```
Comment (rjmackay) [inline app/main/posts/detail/detail.html]: I wonder if this should actually be part of the `post-value` directive instead?

**Comment by rjmackay** [comment]
@Angamanga check out https://github.com/ushahidi/platform-client/pull/732 ... I think it may be fixing the same issue?

**Comment by Angamanga** [comment]
@rjmackay Ohnoe, yes it does. Closing this...

**Pull request closed without merging**